mivs/templates/mivs_applications/game.html

```diff
@@ -0,0 +1,163 @@
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label optional-field">Link to Game:</label>
+        <div class="col-sm-6">
+            <input class="form-control" type="text" name="link_to_game" value="{{ game.link_to_game }}" />
+            <p class="help-block">
+                This does not need to be a public link, but if it's restricted somehow, you must include
+                the password or other instructions for how to access it below.
+            </p>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label optional-field">Download Password:</label>
+        <div class="col-sm-6">
+            <input class="form-control" type="text" name="password_to_game" value="{{ game.password_to_game }}" />
+            <p class="help-block">
+                If access to the above URL is restricted somehow, please enter whatever credentials are
+                necessary to access the game.
+            </p>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Game Activation Code:</label>
+        <div class="col-sm-6">
+            <select name="code_type" class="form-control">
+                {% options c.CODE_TYPE_OPTS game.code_type %}
+            </select>
+            <p class="help-block">
+                Please list any other events where you've already run demos of this game.
+            </p>
+        </div>
+    </div>
+
+    <div class="form-group" style="display: none">
+        <label class="col-sm-3 control-label">Activation Code Instructions:</label>
+        <div class="col-sm-6">
+            <input class="form-control" type="text" name="code_instructions" value="{{ game.code_instructions }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Game Build Status:</label>
+        <div class="col-sm-6">
+            <select name="build_status" class="form-control">
+                {% options c.BUILD_STATUS_OPTS game.build_status %}
+            </select>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label optional-field">Build Notes:</label>
+        <div class="col-sm-6">
+            <input class="form-control" type="text" name="build_notes" value="{{ game.build_notes }}" />
+            <p class="help-block">
+                List anything you already know doesn't work or is buggy.
+            </p>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label optional-field">Demoed At Other Events:</label>
+        <div class="col-sm-6">
+            <textarea class="form-control" name="shown_events" rows="3">{{ game.shown_events }}</textarea>
+            <p class="help-block">
+                Please list any other events where you've already run demos of this game.
+            </p>
+        </div>
+    </div>
+
```

Comment:
As you can see from the verbiage below, I wasn't entirely sure what these two checkboxes should say, so the text now is just placeholder text.